modules/sfm/CMakeLists.txt

```diff
@@ -57,23 +90,11 @@ set(LIBMV_LIGHT_LIBS
 if(Ceres_FOUND)
   add_definitions("-DCERES_FOUND=1")
   list(APPEND LIBMV_LIGHT_LIBS simple_pipeline)
-  list(APPEND LIBMV_LIGHT_INCLUDES "${CERES_INCLUDE_DIR}")
```

Comment:
Might this cause a build failure with ceres 1.14, if that is not in a default include path (assuming retaining compatibility with 1.14 is a goal)? It looks to me like ceres 2.0.0 sets the right target properties w.r.t. the include paths, but ceres 1.14 doesn't seem to do so. Otoh with ceres 2.0.0 `CERES_INCLUDE_DIRS` is not set at all anymore, so you would need to have a conditional to avoid adding an empty include path. In any case, it should be `CERES_INCLUDE_DIRS` not `..._DIR`, I think.

Reply:
Fixed (in sfm in rgbd modules).